.github/workflows/ci-slow.yaml

```diff
@@ -16,9 +16,6 @@ jobs:
           - python-version: '3.12'
             os: macos-latest
             EXTRA: true
-          - python-version: '3.7'
-            os: windows-latest
-            EXTRA: true
```

Comment:
increment to 3.8

Reply:
I changed this to 3.8. Given that even Python 3.10 is reaching it's end-of-life (https://devguide.python.org/versions/) very soon. What about directly jumping to 3.10?